Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -72,6 +72,15 @@
   description: A classic-style weapon infused with advanced atmos technology to allow it to set targets on fire.
   price: 10
 
+- type: uplinkListing
+  id: UplinkSyndieLaserCarbine
+  category: Weapons
+  itemId: WeaponSyndieLaserCarbine
+  listingName: Syndicate Laser Gun
+  description:  A stolen laser gun equipped with several state-of-the-art modifications that allows it to recharge itself slowly. 
+  price: 12
+  surplus: false
```

Comment:
why is this not in surplus?

Reply:
I can remove the non-surplus part, but I was thinking about how nukies could get this thing and an L6 or two C20 bundles, as well as an emag or toolbox by buying the super surplus